crates/db_schema/src/impls/images.rs

```diff
@@ -15,35 +12,55 @@ use diesel::{
   ExpressionMethods,
   NotFound,
   QueryDsl,
-  Table,
 };
 use diesel_async::RunQueryDsl;
 use url::Url;
 
 impl LocalImage {
   pub async fn create(pool: &mut DbPool<'_>, form: &LocalImageForm) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
-    insert_into(local_image)
+    insert_into(local_image::table)
       .values(form)
       .get_result::<Self>(conn)
       .await
   }
 
+  /// This should only be used in the internal API, since it has no page and limit
   pub async fn get_all_by_local_user_id(
     pool: &mut DbPool<'_>,
-    user_id: &LocalUserId,
+    user_id: LocalUserId,
   ) -> Result<Vec<Self>, Error> {
     let conn = &mut get_conn(pool).await?;
-    local_image
-      .filter(local_user_id.eq(user_id))
-      .select(local_image::all_columns())
+    local_image::table
+      .filter(local_image::local_user_id.eq(user_id))
+      .select(local_image::all_columns)
+      .load::<LocalImage>(conn)
+      .await
+  }
```

Comment:
You could get rid of this by using the other one with page: None and limit: None, and calling `limit_and_offset()` in the api handler instead.

Reply:
I combined them in an easier way.